UnityProject/Assets/Scripts/Systems/Inventory/Pickupable.cs

```diff
@@ -213,7 +213,7 @@ private IEnumerator ServerPerformInteractionLogic(HandApply interaction)
 
 	private void PickupAnim(GameObject interactor)
 	{
-		OnMoveToPlayerInventory?.Invoke();
+		OnMoveToPlayerInventory?.Invoke(interactor);
```

Comment:
I get it works, but the event should be invoked in the coroutine that calls this method instead of the method that should be only responsible of playing the picking up animation.

Reply:
done